packages/flutter/test/material/scaffold_test.dart

```diff
@@ -276,12 +276,17 @@ void main() {
     await tester.pumpWidget(
       MediaQuery(
         data: const MediaQueryData(
-          padding: EdgeInsets.only(bottom: 20.0),
+          viewPadding: EdgeInsets.only(bottom: 20.0),
```

Comment:
Currently, the Floating Action Button's position has nothing to do with `padding.bottom`.
Maybe a typo? CC @Piinks : )

Reply:
Likely the case, IIRC this was a big change with a lot of code re-use between tests. Copy paste issue is my guess. 😝